scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -420,7 +420,7 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
 
             $rootScope.publishAfterTansa = function () {
                 if (!onlyTansaProof) {
-                    $scope.saveTopbar().then(continueAfterPublish ? $scope.publishAndContinue : $scope.publish);
+                    $scope.saveTopbar().then(continueAfterPublish === true ? $scope.publishAndContinue : $scope.publish);
```

Comment:
Thanks for the explanation.

This is quite confusing and unexpected. `continueAfterPublish` indicates that it's a bool value. Can we please add a comment here to at least specify what you just said above?

It would be nice to fix this so that bool is bool and item is item. Not either bool or item.